netflix-roulette/src/store/store.ts

```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from "redux";
+import createSagaMiddleware from "redux-saga";
+import { rootSaga } from "./sagas";
+import { rootReducer } from "./reducers";
+
+const sagaMiddleware = createSagaMiddleware();
+
+const store = createStore(rootReducer, applyMiddleware(sagaMiddleware));
```

Comment:
In dev mode, you could've taken advantage of [Redux DevTools](https://redux.js.org/usage/configuring-your-store#integrating-the-devtools-extension) for debugging purposes.